framework/core/less/common/App.less

```diff
@@ -2,8 +2,8 @@
   position: relative !important;
   padding-top: var(--header-height);
   padding-bottom: 50px;
-  overflow-x: hidden;
   min-height: 100vh;
+  max-width: 100vw;
```

Comment:
I prefer moving`overflow-h: hidden` to the body, it doesn't mess with the sticky and continues to prevent horizontal scrolling

Reply:
Woah, I did not know that! Time to edit tens of other projects now, I guess... 🙈